base/doc/ltnews40.tex

```diff
@@ -159,6 +159,20 @@ \section{Introduction}
 
 % To write
 
+\section{Switch to T1 as default encoding in documents using \cs{DocumentMetadata}}
+
+As it is well known the font encoding OT1 supports only 128 characters and 
```

Comment:
About styling: is `\texttt{OT1}` preferable than `OT1` (also `T1`)?

A simple grep/rg shows `\texttt{OT1}` makes up the majority.

<details><summary>Result given by <code>rg --no-heading '\bOT1\b' -g 'ltnews*.tex' base/doc</code></summary>
<p>

```
base/doc/ltnews26.tex:86:128-character encoding \texttt{OT1}.  For Unicode based \TeX\ engines, this
base/doc/ltnews26.tex:95:encoding in addition to the previously supported encodings such as \texttt{OT1}
base/doc/ltnews26.tex:125:default \texttt{OT1} encoding, you may switch encoding in the usual ways, for
base/doc/ltnews26.tex:128:\usepackage[OT1]{fontenc}
base/doc/ltnews07.tex:74:To use these fonts in either T1 or OT1 encoding it is
base/doc/ltnews34.tex:699:However, when using \texttt{OT1} or \texttt{T1} encoded fonts in
base/doc/ltnews34.tex:703:hyphenation results for words containing this ligature (when using the \texttt{OT1} encoding).
base/doc/ltnews33.tex:390:\subsection{More ``dashes'' in encodings \texttt{OT1},
base/doc/ltnews33.tex:405:with \pdfTeX{} (since they are not available in \texttt{OT1} or
base/doc/ltnews04.tex:101:Note that this change does not affect the standard `OT1' Computer
base/doc/ltnews17.tex:83:encodings \texttt{OT1} and \texttt{T1}.
base/doc/ltnews31.tex:504:encoding but in \texttt{OT1} you got a warning on the terminal even
base/doc/ltnews24.tex:174:\textcommabelow{G}).  In the OT1 and T1 encodings the combinations are
base/doc/ltnews24.tex:239:The default font setup in \LaTeXe{} at present is to use the \texttt{OT1}
base/doc/ltnews29.tex:199:encodings \texttt{OT1} and \texttt{T1}.  However, this unfortunately
base/doc/ltnews29.tex:214:     {\usefont{OT1}{cmtt}{m}{n}\asciispace}}
base/doc/ltnews15.tex:113:in both the \texttt{OT1} and \texttt{T1} encodings.
```

</p>
</details>